mempool/metrics.go

```diff
@@ -86,23 +95,39 @@ func PrometheusMetrics(namespace string, labelsAndValues ...string) *Metrics {
 			Help:      "Number of evicted transactions.",
 		}, labels).With(labelsAndValues...),
 
+		SuccessfulTxs: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "successful_txs",
+			Help:      "Number of times transactions successfully make it into a block.",
+		}, labels).With(labelsAndValues...),
+
 		RecheckTimes: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
 			Namespace: namespace,
 			Subsystem: MetricsSubsystem,
 			Name:      "recheck_times",
 			Help:      "Number of times transactions are rechecked in the mempool.",
 		}, labels).With(labelsAndValues...),
+
+		AlreadySeenTxs: prometheus.NewCounterFrom(stdprometheus.CounterOpts{
+			Namespace: namespace,
+			Subsystem: MetricsSubsystem,
+			Name:      "already_seen_txs",
+			Help:      "Number of times transactions that try to enter a mempool but already exist.",
```

Comment:
[nit] so this help text more closely matches the doc comment above `AlreadySeenTxs`

```suggestion
			Help:      "Number of transactions that entered the mempool but were already present in the mempool.",
```